warehouse/templates/base.html

```diff
@@ -135,11 +135,12 @@
     </div>
     {% endfor %}
 
-    {% block session_notifications %}{% endblock %}
-
-    <div class="notification-bar">
-      <span class="notification-bar__message">Help us improve Python packaging – <a href="https://donate.pypi.org">donate today!</a></span>
-    </div>
+    {% block session_notifications %}
+      <div id="user-research" data-version="0" class="notification-bar notification-bar--warning notification-bar--dismissable" data-controller="notification" data-target="notification.notification">
+        <button title="Dismiss this notification" data-target="notification.dismissButton" data-action="click->notification#dismiss" class="notification-bar__dismiss" aria-label="close"><i class="fa fa-times" aria-hidden="true"></i></button>
+        <span class="notification-bar__message">Help us improve the PyPI user interface by participating in <a href="https://t.co/aYpp2ve2iY">5 minute user research</a>!</span>
```

Comment:
> Help us improve the PyPI user interface by participating in 5 minute user research

This reads a little funny to me, feels like it's missing a word. Perhaps one of:

> Help improve the PyPI user interface by participating in a 5 minute user research survey
> Help improve PyPI by participating in a 5-minute user interface survey
> Help improve PyPI by participating in a 5-minute UI/UX survey

would be better?

Reply:
I like "Help improve PyPI by participating in a 5-minute user interface survey", I'll push an update :)